src/components/KSlideout/KSlideout.vue

```diff
@@ -48,6 +50,24 @@ const props = defineProps({
     type: Boolean,
     default: false,
   },
+  // controls close button alignment
+  closeBtnAlignment: {
+    type: String,
+    default: 'start',
+    validator: (value: string): boolean => {
+      return ['start', 'end'].includes(value)
+    },
+  },
+  // enable/disable overlay to be able to interact with the background content while the slideout is expanded
+  overlayEnabled: {
+    type: Boolean,
+    default: false,
+  },
+  // allows a host app to define the offset from the top of the page
+  topOffsetNumeric: {
```

Comment:
can this just be named `offsetTop`?

Also it doesn't match the name of the prop in the docs